### ISSUE TITLE
[Keyboard navigation - Make code Arcade- Teach]: 'Start video' button is not keyboard accessible.

### ISSUE THREAD

**Issue by vroopea** [microsoft/pxt-microbit#6101]
### User Experience:
Keyboard dependent people will get impacted if any interactive element present on the page is not keyboard accessible. Due to which the end users will not be able to access the video of the bring code into your classroom section and might face difficulty in accessing the page.

**Note:**
User credentials should NOT be included in the bug.

### Repro Steps:

1. Launch application URL: https://www.microsoft.com/en-us/makecode/teach
2. Teach page will open.
3. Navigate the page using tab key and observe whether 'Start video' button is keyboard accessible or not.

### Actual Result:
**Issue:** 'Start video' button is not keyboard accessible as it was not getting activated with 'Enter/Space/ CTRL Enter.

**Observation** There is play button beside the 'Start video' control, but keyboard focus doesn't move to it while navigating using tab key.

### Expected Result:
'Start video' button should be keyboard accessible. When focus is on the 'Start video' button, by pressing 'Enter/Space' it should get activated.

### MAS Reference:
[MAS 2.1.1 – Keyboard](https://aka.ms/MAS2.1.1)

### ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:
NA

### UIA/DOM details available:
NA

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:
**OS Version:** Windows 11 24H2 (OS Build:26100.3037)
**Browser:** Chrome Version 132.0.6834.196 (Official Build) (64-bit)
**Test Matrix:** Chrome + NVDA
**URL:** https://www.microsoft.com/en-us/makecode/teach

### Attachment:

https://github.com/user-attachments/assets/859943d6-9847-4f0e-922b-7e9f64809227

**Comment by vroopea** [comment]
#Accessibility; #A11yMAS; #A11ySev1; #MAS2.1.1; #AILmited; #HCL; #Benchmark; #Win11-Chrome; #BM-HCL-MakeCode-Feb2025

**Comment by abchatra** [comment]
This is external site owned by microsoft.com and not makecode.

**Comment by Yamini234u** [comment]
As checked issue on  https://www.microsoft.com/en-us/makecode/teach. The issue has been fixed. Hence, closing the bug

**Comment by Yamini234u** [comment]
As checked issue on https://www.microsoft.com/en-us/makecode/teach. The issue has been fixed. Hence, closing the bug
